nodejs/week3/src/backend/api/meals.js

```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const pool = require("./../database");
+
+// Returns all meals	
+router.get("/", (request, response) => {
+  pool.query("SELECT * FROM meal", function(error, results) {
+    if (error) {
+      console.log(error);
+      response.send("No meals were found, please check message error")
+    } else {
+    return response.json(results);
+    
+} next();
```

Comment:
`next` would crash if you ever reached this code. It's still after `if/else` blocks, so you will never have it executed. Could you fix it? Also, the alignment is broken in a lot of places. If you fix it, the code will become much more readable.

Reply:
Corrected in PR https://github.com/Lucha2040/hyf-homework/pull/36/commits/586da540138991a0d004e8f01f601105b8d20d4d
But some routing issues persist.